lambda/js/src/main/scala/feral/lambda/IOLambdaPlatform.scala

```diff
@@ -26,12 +26,23 @@ import scala.scalajs.js.|
 private[lambda] trait IOLambdaPlatform[Event, Result] {
   this: IOLambda[Event, Result] =>
 
+  /**
+   * Lambda handler. Implement this type with a val and a call to `handlerFn` to export your
+   * handler.
+   *
+   * @example
+   *   {{{
+   *  val handler: HandlerFn = handlerFn
+   *   }}}
+   */
+  final type HandlerFn = js.Function2[js.Any, facade.Context, js.Promise[js.Any | Unit]]
+
   final def main(args: Array[String]): Unit =
     js.Dynamic.global.exports.updateDynamic(handlerName)(handlerFn)
```

Comment:
I guess we can remove this now :)

Reply:
I can remove it if you want, but it would be a breaking change for any existing users. It might be nice to still have the old implementation for CommonJS users. Maybe we can also add a warning or clear error when this is called inside ESModule

Reply:
I have to think about this. I think I dislike the idea that there are two distinct ways to define JS Lambdas, and "dead" code depending on which you choose (such as `handlerName` for ESModules). I think this would be very confusing. Since your approach words for both, I'd rather buy-in all the way.

Reply:
Alright, I suppose a breaking change like this is better to do early in the life of a library. If I have your permission, I'll remove the related code

Reply:
Exactly. Please go ahead with the change, although maybe we should make this PR the beginning of an v0.2.x series since this is a significant change (ignoring the fact that 0.1.x hasn't materialized a stable release 😅). You can bump the `tlBaseVersion` in the build to signify this.

Reply:
Done. I don't think I've missed anything, but you are welcome to double-check me :)

Reply:
Thanks! I'm afraid the example project will also need updating :) To keep it cross-compiling with JVM I think we can add a `@JSExport` stub.
https://github.com/scala-js/scala-js-stubs/